From 0a6b65022eb147326d5592762040b3b6355cef99 Mon Sep 17 00:00:00 2001
From: nrndda <nrndda@gmail.com>
Date: Sat, 26 Jun 2021 08:05:31 +0300
Subject: [PATCH] Fix build with VAAPI >=1.0

---
 src/media_drv_init.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/media_drv_init.c b/src/media_drv_init.c
index 36ddce4..62c374e 100644
--- a/src/media_drv_init.c
+++ b/src/media_drv_init.c
@@ -2764,9 +2764,9 @@ va_driver_init (VADriverContextP ctx)
   return ret;
 }
 
-VAStatus DLL_EXPORT __vaDriverInit_0_34 (VADriverContextP ctx);
+VAStatus DLL_EXPORT VA_DRIVER_INIT_FUNC (VADriverContextP ctx);
 VAStatus
-__vaDriverInit_0_34 (VADriverContextP ctx)
+VA_DRIVER_INIT_FUNC (VADriverContextP ctx)
 {
   VAStatus ret = VA_STATUS_ERROR_UNKNOWN;
 
